packages/jest-config/src/ValidConfig.ts

```diff
@@ -66,6 +66,10 @@ const initialOptions: Config.InitialOptions = {
     throwOnModuleCollision: false,
   },
   injectGlobals: true,
+  inlineSnapshotFormatter: multipleValidOptions(
+    'exposed prototypes',
+    'simple',
+  ) as any,
```

Comment:
It was either throw an `as any` here or do some type foo to support making those options get turned into a matching union